app/src/main/java/org/mozilla/fenix/loginexceptions/LoginExceptionsFragment.kt

```diff
@@ -0,0 +1,88 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.loginexceptions
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.lifecycle.asLiveData
+import androidx.lifecycle.lifecycleScope
+import kotlinx.android.synthetic.main.fragment_exceptions.view.*
+import kotlinx.coroutines.Dispatchers.IO
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.launch
+import mozilla.components.feature.logins.exceptions.LoginException
+import mozilla.components.lib.state.ext.consumeFrom
+import org.mozilla.fenix.R
+import org.mozilla.fenix.components.StoreProvider
+import org.mozilla.fenix.ext.requireComponents
+import org.mozilla.fenix.ext.showToolbar
+
+/**
+ * Displays a list of sites that are exempted from saving logins,
+ * along with controls to remove the exception.
+ */
+class LoginExceptionsFragment : Fragment() {
+    private lateinit var exceptionsStore: ExceptionsFragmentStore
+    private lateinit var exceptionsView: LoginExceptionsView
+    private lateinit var exceptionsInteractor: LoginExceptionsInteractor
+
+    override fun onResume() {
+        super.onResume()
+        showToolbar(getString(R.string.preference_exceptions))
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val view = inflater.inflate(R.layout.fragment_exceptions, container, false)
```

Comment:
This is actually used by both the TrackingProtectionExceptions and LoginExceptions because in theory these screens should look very similar. I was trying to reduce _some_ of the duplication by sharing some layout files here and was going to file a ticket to try to reuse more later

Reply:
Opened https://github.com/mozilla-mobile/fenix/issues/12522